conf/simulator/jsbsim/aircraft/rotwingv3c_SI.xml

```diff
@@ -0,0 +1,629 @@
+    <property>fcs/right_motor</property>
+    <property>fcs/right_motor_lag</property>
+    <property>fcs/right_motor_d</property>
+    <property>fcs/back_motor</property>
+    <property>fcs/back_motor_lag</property>
+    <property>fcs/back_motor_d</property>
+    <property>fcs/left_motor</property>
+    <property>fcs/left_motor_lag</property>
+    <property>fcs/left_motor_d</property>
+    <!-- <property>fcs/rudder</property>
+    <property>fcs/elevator</property>
+    <property>fcs/aileron</property>-->
+    <property>fcs/pusher</property>
+    <property>fcs/pusher_lag</property>
+     <!-- <property>fcs/flap</property>
+    <property>fcs/skew</property> -->
+
+    <!-- First the lift forces produced by each propeller -->
+
+    <force name="front_motor" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/front_motor_lag</property>
+          <value> 8.69 </value>
+        </product>
+      </function>
+      <location unit="M">
+        <x>-0.423</x>
+        <y>0</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>0</x>
+        <y>0</y>
+        <z>-1</z>
+      </direction>
+    </force>
+
+    <force name="right_motor" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/right_motor_lag</property>
+          <value> 8.69 </value>
+        </product>
+      </function>
+      <location unit="M">
+        <x>0</x>
+        <y>0.408</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>0</x>
+        <y>0</y>
+        <z>-1</z>
+      </direction>
+    </force>
+
+    <force name="back_motor" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/back_motor_lag</property>
+          <value> 8.69 </value>
+        </product>
+      </function>
+      <location unit="M">
+        <x>0.423</x>
+        <y>0</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>0</x>
+        <y>0</y>
+        <z>-1</z>
+      </direction>
+    </force>
+
+    <force name="left_motor" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/left_motor_lag</property>
+          <value> 8.69 </value>
+        </product>
+      </function>
+      <location unit="M">
+        <x>0</x>
+        <y>-0.408</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>0</x>
+        <y>0</y>
+        <z>-1</z>
+      </direction>
+    </force>
+
+    <force name="pusher_motor" frame="BODY" unit="N">
+      <function>
+        <product>
+          <property>fcs/pusher_lag</property>
+          <value> 36.96 </value>
+        </product>
+      </function>
+      <location unit="M">
+        <x>0</x>
+        <y>0</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>1</x>
+        <y>0</y>
+        <z>0</z>
+      </direction>
+    </force>
+
+    <!-- Now the moment couples -->
+    <force name="front_motor_couple1" frame="BODY" unit="N">
```

Comment:
It is now possible to define moments directly instead of using two forces: see example with https://github.com/paparazzi/paparazzi/blob/master/conf/simulator/jsbsim/aircraft/simple_quad_wind.xml#L214

Reply:
Thank you for the advice! Indeed using the moment function directly is so much easier and also understandable.